envs/example/defaults.yml

```diff
@@ -107,8 +107,22 @@ nova:
     debug: True
     verbose: True
 
+ceph:
+  enabled: false
+  rbd_pool: default
+  rbd_user: cinder
+  rbd_ceph_conf: /etc/ceph/ceph.conf
+  rbd_flatten_volume_from_snapshot: false
+  rbd_max_clone_depth: 5
+  glance_api_version: 2
+
 cinder:
   enabled: True
+  enabled_backends: #RBD_VOLUMES for ceph
+  backends: []
+    # Ceph
+    #- name: RBD_VOLUMES
```

Comment:
I discovered an issue in cinder where naming the backend config blog with uppercase characters can lead to mismatches between what the cinder-volume host thinks it is and what the database entry is and what the scheduler thinks it is. The solution is to make sure the config block is all lowercase, so the choice of RDB_VOLUMES conflicts with this.